java-client/session/src/main/java/io/deephaven/client/impl/BatchTableRequestBuilder.java

```diff
@@ -236,6 +238,15 @@ public Operation visit(TailTable tailTable) {
                     .setSourceId(ref(tailTable.parent())).setNumRows(tailTable.size()));
         }
 
+        @Override
+        public Operation visit(SliceTable sliceTable) {
+            return op(Builder::setSlice, SliceRequest.newBuilder().setResultId(ticket)
+                    .setSourceId(ref(sliceTable.parent()))
+                    .setFirstPositionInclusive(sliceTable.firstPositionInclusive())
+                    .setLastPositionExclusive(sliceTable.lastPositionExclusive())
+                    .build());
+        }
```

Comment:
Can you add a java client test to make sure things succeed end to end? `java-client/session-dagger/src/test/java/io/deephaven/client/MultiJoinTest.java` is the pattern I would follow.